README.md

```diff
@@ -11,29 +11,74 @@ theory behind these constructors is researched in
 [DOT](http://www.cs.uwm.edu/~boyland/fool2012/papers/fool2012_submission_3.pdf),
 a calculus for dependent object types.
 
-The dotty compiler is largely a new design. It takes a more functional
-approach than current scalac, paired with aggressive caching to
-achieve good performance. Some parts, in particular those that have to
-do with configuration and input/output are ported from the Scala
-compiler. The compiler is fully functional, in particular it can
-compile itself. But there's more work needed (and planned) on
-polishing rough edges, improving diagnostics, improving compilation
-speed, and embedding in other tools.
-
-We expect that, over time, some of the new technologies explored in
-this project will find their way into future versions of Scala. At
-present it is too early to say which ones and when.
-
-If you want to try it out, to get started have a look at
+####Current status:
+ _Technology preview_: currently unsupported, may not be functionally complete, and not suitable for deployment in production.
+
+####Is it going to be the future Scala?
+Yes, eventually.
+
+####Who's working on it?
+See [github contributors page](https://github.com/lampepfl/dotty/graphs/contributors).
+ 
+####What are the features that could make me consider trying it?  
+| Feature                                                                                                 | Status              |
+|---------------------------------------------------------------------------------------------------------|---------------------|
+| Union, Intersection and [Literal singleton types](http://docs.scala-lang.org/sips/pending/42.type.html) | Implemented         |
+| Fast compilation (phase fusion)                                                                         | Implemented         |
+| [Trait parameters](http://docs.scala-lang.org/sips/pending/trait-parameters.html)                       | Implemented         |
+| [@Static methods and fields](https://github.com/scala/scala.github.com/pull/491)                        | Implemented         |
+| Colored Repl                                                                                            | Implemented         |
+| Sbt incremental build                                                                                   | Implemented         |
+| Non-blocking lazy vals                                                                                  | Implemented         |
```

Comment:
There's an SIP for that but I don't know if it matches what's in Dotty: http://docs.scala-lang.org/sips/pending/improved-lazy-val-initialization.html